test/test_external.py

```diff
@@ -39,6 +39,7 @@
 
 class TestLoadInterface:
     def test_audio_to_audio(self):
+        assert False
```

Comment:
Will revert prior to merge. Just to show that the non-flaky tests finish running.